app/models/user.js

```diff
@@ -27,6 +27,12 @@ module.exports = (sequelize, DataTypes) => {
         type: DataTypes.ENUM,
         values: ['regular', 'admin'],
         defaultValue: 'regular'
+      },
+      allowedDate: {
+        allowNull: false,
+        type: DataTypes.DATE,
+        field: 'allowed_date',
+        defaultValue: Date.now()
```

Comment:
This value `Date.now()` its constant. Check how to use literal function for save, for example `NOW()` or `SYSDATE()`